src/components/Integrations/BigQuery/AuthorizationPage.test.js

```diff
@@ -10,30 +10,37 @@ describe('test for validate authorization form component ', () => {
     emails: ['email1@gmail.com', 'email2@gmail.com', 'email3@gmail.com'],
   };
 
-  const bigQueryClientDouble = {
+  const bigQueryClientDouble = (success) => ({
```

Comment:
can you remove `logRoles` method in AuthorizationPage.test?  It's not used
![image](https://user-images.githubusercontent.com/84402180/131348844-4d4b2b17-5873-4bb7-8402-f2ff6b19019c.png)

Reply:
suggest accepted and applied